src/CMakeLists.txt

```diff
@@ -222,6 +233,16 @@ if(Omega_h_USE_SimModSuite)
   target_link_libraries(omega_h PUBLIC "${SIMMODSUITE_LIBS}")
 endif()
 
+if(Omega_h_USE_ADIOS2)
+  set(ADIOS_REQUIRED_VERSION 2.10)
+  if (Omega_h_USE_MPI)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DADIOS2_USE_MPI")
+    target_include_directories(omega_h PUBLIC "${ADIOS2_INCLUDE_DIR}")
+    target_link_libraries(omega_h PUBLIC "${ADIOS2_LIBS}")
```

Comment:
I think the target should be `adios2::cxx11_mpi` see: https://github.com/SCOREC/redev/blob/f4b1d75b6996f5ed0a7c87c25f9d146d22babc80/CMakeLists.txt#L54